docs/api/wanchain.md

```diff
@@ -2079,9 +2126,9 @@ This order kind can perform multiple operations such as value transfer, asset tr
 | signature | [required] A `string` representing order signature created by the maker.
 | order.actions | [required] An `array` of [actions order action objects](#actions order-actions).
 | order.expiration | [required] An `integer` number representing the timestamp in milliseconds after which the order expires and can not be performed any more.
-| order.makerId | [required] A `string` representing an Ethereum account address which makes the order. It defaults to the `accountId` of a provider.
+| order.makerId | [required] A `string` representing an Wanchain account address which makes the order. It defaults to the `accountId` of a provider.
```

Comment:
| order.makerId | [required] A `string` representing a Wanchain account address which makes the order. It defaults to the `accountId` of a provider.